### PR TITLE
Add smoke test for JS bundle

### LINKED ISSUE #277: Scopes built on circle won't load UI
```
Uncaught Error: Cannot find module "../../node_modules/materialize-css/font/roboto/Roboto-Regular.woff2"
```

### PR DESCRIPTION
**Pull request by davkal**
run JS smoke test before make static
fixes #277

### CODE REVIEW

File: client/test/smoke-bundle.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+page.onError = function(msg, trace) {
+  console.error(msg);
+  phantom.exit(1);
+};
+
+page.open(url, function (status) {
+  console.log(status);
+  //Page is loaded!
+  phantom.exit();
+});
```
Comment (tomwilkie) [inline client/test/smoke-bundle.js]: newline

File: circle.yml [hunk 2]
```diff
@@ -35,10 +35,10 @@ dependencies:
 test:
   override:
     - cd $SRCDIR; ./bin/lint .
+    - cd $SRCDIR; make client-test
```
Comment (tomwilkie) [inline circle.yml]: Why is this moved?

Comment (davkal) [inline circle.yml]: test time <<< build time
and the build is not needed for testing

**Comment by tomwilkie** [comment]
LGTM
